### DISCUSSION TITLE
Разделение syslog на нужный и debug

### DISCUSSION THREAD

**Discussion by subs1stem** [rosteleset/SmartYard-Server#81] [Ideas]
Предлагаю разделить поток syslog сообщений на две таблички. В `syslog` писать только "жёстко" зафильтрованные сообщения, касающиеся работы непосредственно домофонной части и нужные для plog (ключи, коды, открытия дверей, звонки и т.д.). В какую-нибудь табличку `debug` писать абсолютно всё и пусть оно живёт дней 5 например. Сейчас в syslog всё равно проникает куча ненужного мусора, но когда возникает необходимость для отладки что-то посмотреть, то часть сообщений отфильтрована (о том, что оно отфильтровалось, ещё частенько забываешь).

**Comment by mmikel** [comment]
скорее уж добавить поле "level" или еще как-то, clickhouse'у по барабану, он сотни тысяч инсертов в секунду пережевывает спокойно... clickhouse это все-таки аналитическая БД, он рассчитан на то что "всех впускать"...
